canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -229,32 +229,32 @@ $toolbar-divider-width: 1px;
 	}
 
 	& .toolbar-up-down-chevron {
-		padding: 11px 8px 11px 0;
+		padding: $spacing-04 $spacing-03 $spacing-04 0;
```

Comment:
this can be replaced with `display:flex` and `align-items:center`

Reply:
For some reason `display: flex` and `align-items: center` did not change much, so I reverted the changes

Reply:
Those styles need to apply to its parent